Development/Porting.md

```diff
@@ -5,86 +5,220 @@ permalink: /Porting/
 nav_order: 3
 parent: Development
 ---
-TODO: UPDATE ME!
 
-To add a new board to the Heads build hopefully you only need to modify
-the coreboot configuration and add a top-level image configuration.
+<!-- markdownlint-disable MD033 -->
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+<!-- markdownlint-enable MD033 -->
 
-* Copy one of the existing image configurations, such as `config/x230-qubes.config`
-to your new image, `config/newarch-qubes.config`, and edit the file to change
-`BOARD=x230` to `BOARD=newarch`.
+Prerequisites for porting Heads
+===
+Prerequisites: 
+1. coreboot port:  since Heads is a coreboot payload, the board must have fully completed and actively maintained coreboot support. Any known issues should be acceptable to end users. Exceptions include the Librem and Dasharo coreboot forks, where coreboot is used from their respective forks.
```

Comment:
Exceptions include the Purism and Dasharo coreboot forks, where coreboot is used from their respective forks.